packages/sources/vue/src/components/overlays/VtmnToast/VtmnToast.vue

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+import '@vtmn/css-toast/dist/index-with-vars.css';
+import { reactive, computed, defineComponent, PropType } from 'vue';
+import { VtmnButton } from '../../index';
+
+export default /*#__PURE__*/ defineComponent({
+  name: 'VtmnToast',
+  inheritAttrs: false,
+  components: { VtmnButton },
+  props: {
+    withIcon: {
```

Comment:
Don't forget "as PropType...." for all props (for TS better integration

Reply:
ok, thanks